buildspec/windowsTests.yml

```diff
@@ -20,7 +20,7 @@ phases:
 
   build:
     commands:
-      - ./gradlew check coverageReport --info --full-stacktrace --console plain
+      - ./gradlew :jetbrains-rider:check --info --full-stacktrace --console plain
```

Comment:
Why are we only building rider on windows?

Reply:
Sorry WIP PR that I forgot to mark

Reply:
Trying to get this PR to build on top of #1613. If I can get all tests to pass here, I will merge the other PR and then merge this right after